test/suite/test_hs01.py

```diff
@@ -97,14 +99,14 @@ def durable_check(self, check_value, uri, ds, nrows):
         conn.close()
 
     def test_hs(self):
-        # Create a small table.
+        # Create a small table with 100 rows.
         uri = "table:test_hs01"
         nrows = 100
         ds = SimpleDataSet(self, uri, nrows, key_format=self.key_format, value_format='u')
         ds.populate()
-        bigvalue = b"aaaaa" * 100
 
-        # Initially load huge data.
+        # Initially insert a huge number of data.
```

Comment:
This is grammatically incorrect, probably just say "Initially insert a lot of data."